app/build.gradle

```diff
@@ -62,29 +62,32 @@ dependencies {
     compile "com.android.support:cardview-v7:$androidSupportVersion"
     compile "com.android.support:design:$androidSupportVersion"
     compile "com.android.support:preference-v7:$androidSupportVersion"
-    compile 'com.google.android.gms:play-services-gcm:10.2.1'
-    compile 'com.google.code.gson:gson:2.8.0'
-    compile 'com.google.guava:guava:20.0'
-    compile 'se.emilsjolander:stickylistheaders:2.7.0'
     compile "com.squareup.retrofit2:retrofit:$retrofitVersion"
     compile "com.squareup.retrofit2:converter-gson:$retrofitVersion"
-    compile 'com.github.chrisbanes:PhotoView:1.3.0'
-    compile 'me.dm7.barcodescanner:zxing:1.9'
     compile('org.simpleframework:simple-xml:2.7.1') {
         exclude group: 'stax', module: 'stax-api'
         exclude group: 'xpp3', module: 'xpp3'
     }
+
+    androidTestCompile "com.android.support:support-annotations:$androidSupportVersion"
+    androidTestCompile "com.android.support.test.espresso:espresso-core:$espressoVersion"
+    androidTestCompile "com.android.support.test.espresso:espresso-contrib:$espressoVersion"
+
+    androidTestCompile "com.android.support:support-annotations:$androidSupportVersion"
+    compile 'com.google.android.gms:play-services-gcm:10.2.1'
+    compile 'com.google.code.gson:gson:2.8.0'
+    compile 'com.google.guava:guava:20.0'
+    compile 'se.emilsjolander:stickylistheaders:2.7.0'
+    compile 'com.github.chrisbanes:PhotoView:1.3.0'
+    compile 'me.dm7.barcodescanner:zxing:1.9'
     compile 'de.psdev.licensesdialog:licensesdialog:1.8.2'
     compile 'com.github.alamkanak:android-week-view:1.2.6'
     compile 'de.hdodenhof:circleimageview:2.1.0'
     compile 'net.danlew:android.joda:2.9.9'
     compile 'com.github.franmontiel:PersistentCookieJar:v1.0.0'
     compile 'com.github.barteksc:android-pdf-viewer:2.5.1'
-
-    androidTestCompile "com.android.support:support-annotations:$androidSupportVersion"
+    compile 'com.android.support.constraint:constraint-layout:1.0.2'
     androidTestCompile 'com.android.support.test:runner:0.5'
     androidTestCompile 'com.android.support.test:rules:0.5'
     androidTestCompile 'org.hamcrest:hamcrest-library:1.3'
-    androidTestCompile "com.android.support.test.espresso:espresso-core:$espressoVersion"
-    androidTestCompile "com.android.support.test.espresso:espresso-contrib:$espressoVersion"
 }
```

Comment:
yes, please remove this. This file should not be touched unless you add dependencies.